.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
+	path = shared-modules
+	url = https://github.com/flathub/shared-modules.git
```

Comment:
shared-modules isn't added properly

Reply:
Sorry. It is my first time working with submodules. `git` created this file when I added the submodule. Unfortunately, I don't see what is wrong. I also checked https://github.com/flathub/org.octave.Octave/blob/master/.gitmodules but it looks identical. Do you know what is missing?

Reply:
You have to add and commit the shared-modules folder.

Reply:
Sorry, that was my fault. I added the submodules folder to the `.gitignore` file :see_no_evil:. Fixed in https://github.com/AlbrechtL/flathub/commit/114d64d7ea2373d7322226c48fc042fe3ad638a1.